nss-day-cohort-39/hello-world-where-do-we-want-to-go
# Resolving conflicts with master, added lists of cites, landmarks, and heads of state to italty.html

Changes that were made:

-Fixed conflicts with master
-Added italy.html
-Added list of cities*4, landmarks*4, and heads of state*2 in index.html
-Added Italy image to index.html

Steps to test:

1. Make sure I did not break master
2. Click Italy in nav bar and ensure that it navigate to Italy page
3. Make sure you see cities*4, landmarks*4, and heads of state*2 at bottom of Italy page
4. Make sure you see Italy image under nav bar on Italy page
5. Make sure you are still able to navigate to france page from nav bar
